R/import_data.R

```diff
@@ -277,7 +276,7 @@ iNZread.dta <- function(obj, ...) {
                                        convert.dates   = TRUE,
                                        convert.factors = TRUE)
   
-  attr(temp.data.frame, "full.file") = TRUE
+  attr(temp.data.frame, "preview") = obj$preview
```

Comment:
Yes, that is why I wanted to have 2 different vairables, preview and full.file. 

Do you suggest that I put preview as FALSE if the function is incapable of reading only part of a file irrespective of what the role of the reading was i.e preview or final read ?